Sources/tart/Commands/Push.swift

```diff
@@ -54,48 +54,43 @@ struct Push: AsyncParsableCommand {
         + "\(registryIdentifier.host)/\(registryIdentifier.namespace)\(remoteNamesForRegistry.referenceNames())...")
 
       let references = remoteNamesForRegistry.map{ $0.reference.value }
-      let pushedRemoteName: RemoteName
-      let lightweightPush: Bool
 
+      let pushedRemoteName: RemoteName
       // If we're pushing a local OCI VM, check if points to an already existing registry manifest
       // and if so, only upload manifests (without config, disk and NVRAM) to the user-specified references
-      if let remoteName = try? RemoteName(localName),
-         let remoteName = try await lightweightPushToRegistry(registry: registry, remoteName: remoteName,
-                                                              references: references) {
-        pushedRemoteName = remoteName
-        lightweightPush = true
+      if let remoteName = try? RemoteName(localName) {
+        pushedRemoteName = try await lightweightPushToRegistry(
```

Comment:
Imagine we have an OCI image that is not present in the registry. Because we don't have an additional condition as we have in https://github.com/cirruslabs/tart/pull/465 before jumping into this branch, `tart push` will be of no use for uploading that OCI image to the registry.

Reply:
I think it's a reasonable assumption. If you have an OCI image locally, you pulled it from somewhere. If it's missing remotely, the layers might miss as well and it's reasonable to fail here IMO.

Reply:
>If it's missing remotely, the layers might miss as well and it's reasonable to fail here IMO.

Sorry, but I don't think it is reasonable at all: your proposal essentially prevents the users from uploading these layers again to the registry in the event of their deletion on the registry's side.